content/reference/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2017-05-29
+
+  * [doc update] Add docs for `applications` endpoint.
```

Comment:
Its all mixed up in the changelog tbh.
Latest addition of endpoint is labeled as `doc update`